doc/package.json

```diff
@@ -19,6 +19,7 @@
       "sourceType": "module"
     }
   },
+  "exports": {},
```

Comment:
This is a hack for now given that [typescript docs verifier](https://github.com/bbc/typescript-docs-verifier/blob/30a2d634b7cae88b68d8cbcbcb873b222ea3b732/src/PackageInfo.ts#L69) relies on the exports for its check